notebook_review_process.md

```diff
@@ -28,8 +28,14 @@ For authors: consider these checklists requirements for your code.
    	- Do all code cells have corresponding narratives/comments?
    	- Include information about which server type and environment to choose when logging in to Fornax and the notebook's expected runtime given that setup. For example: 
    	- "As of 2024 August, this notebook takes about 3 minutes to run to completion on Fornax using Server Type: 'Standard - 8GB RAM/4 CPU' and Environment: 'Default Astrophysics' (image)."
+- Dependencies and imports:
+        - Does the notebook have a corresponding `requirements_<notebook_filename>.txt` file listing all its direct dependencies?
+        - Are all dependencies listed in the requirements file in fact required? Do revisit the list as the notebook evolve.
+        - Is the requirements file used in a commented-out cell in the notebook  with `pip install -r <filename>`; and has the notebook no onther installation related cells?
+        - Are dependencies kept to a minimum? E.g. no new library introduced for minimal usage while another library that is already a dependency can do the same functionality?
+          (e.g. do not introduce pandas to print out a line from a FITS table for which we already need to use astropy for IO; add dependencies when their functionality is unique or required for efficiency, etc.)
+        - Are all imports used? Do not import functionality, modules, and packages that are not used in the notebook.
```

Comment:
May want to consolidate this item with the one below in the Cleanup section, "Have unused libraries been removed from the requirements.txt file and the `import` statements?" and the one above about the requirements file.

Reply:
Indeed, it's there already.